src/conversions.jl

```diff
@@ -591,6 +591,43 @@ function cal2jd(iy, im, id)
     jd, jd1
 end
 
+function dtf2d(scale, iy, im, id, ihr, imn, sec)
+    jd, w = cal2jd(iy, im, id)
+    jd += w
+    seclim = 60.0
+
+    if scale == "UTC"
```

Comment:
The `scale` parameter should be a `TimeScale` and not a `String`.

Reply:
Would make it sense to test if it is a CoordinatedUniversalTime? Or the compiler is already smart enough to simplify everything?

Reply:
Seems to be smart enough. Both variants generate the same LLVM IR.

Reply:
Great!